src/components/accordion/accordion.style.ts

```diff
@@ -59,14 +81,79 @@ const StyledAccordionContainer = styled.div<StyledAccordionContainerProps>`
         border-bottom: 1px solid var(--colorsUtilityMajor100);
       }
     `}
+
+  &:focus {
+    ${addFocusStyling()}
+  }
 `;
 
-interface StyledAccordionTitleProps {
-  size?: "large" | "small";
-  variant?: "standard" | "subtle";
-}
+const StyledDetails = styled.details<StyledDetailsProps>`
+  overflow: hidden;
+
+  &[open] + div.content {
+    max-height: 800px;
+    padding-top: ${({ disableContentPadding, size, variant }) =>
+      disableContentPadding
+        ? "0"
+        : size === "small"
+          ? "var(--spacing200)"
+          : variant === "subtle"
+            ? "var(--spacing100)"
+            : "var(--spacing300)"};
+    padding-bottom: ${({ disableContentPadding, size }) =>
+      disableContentPadding
+        ? "0"
+        : size === "small"
+          ? "var(--spacing200)"
+          : "var(--spacing300)"};
+    /* istanbul ignore next */
+    transition:
+      max-height
+        ${({ reduceMotion }) =>
+          getTransitionMs(reduceMotion, defaultMaxHeightTransitionMs)}
+        ease-in-out,
+      padding
+        ${({ reduceMotion }) =>
+          getTransitionMs(reduceMotion, defaultPaddingTransitionMs)}
+        linear;
+  }
+
+  span {
+    rotate: 0deg;
+    /* istanbul ignore next */
+    transition: rotate
+      ${({ reduceMotion }) =>
+        getTransitionMs(reduceMotion, defaultRotateTransitionMs)}
+      ease-in;
+  }
+
+  &[open] span {
+    rotate: ${({ iconAlign }) =>
+      iconAlign === "right" ? "180deg" : "-180deg"};
+    /* istanbul ignore next */
+    transition: rotate
+      ${({ reduceMotion }) =>
+        getTransitionMs(reduceMotion, defaultRotateTransitionMs)}
+      ease-out;
+  }
+`;
 
-const StyledAccordionTitle = styled.h3<StyledAccordionTitleProps>`
+const StyledSummary = styled.summary`
+  ${space}
+  cursor: pointer;
+  list-style: none;
+  user-select: none;
+  ::-webkit-details-marker {
+    display: none;
+  }
+  ::marker {
+    display: none;
+  }
+`;
+
+const StyledSummaryTitleWrapper = styled(Box)``;
```

Comment:
question: do we need this, could we not just use Box in the component file instead?